pkgs/development/tools/build-managers/buildbot/default.nix

```diff
@@ -57,12 +57,13 @@ pythonPackages.buildPythonApplication (rec {
   preInstall = ''
     # writes out a file that can't be read properly
     sed -i.bak -e '69,84d' buildbot/test/unit/test_www_config.py
-
-    # re-hardcode path to tail
-    sed -i.bak 's|/usr/bin/tail|${coreutils}/bin/tail|' buildbot/scripts/logwatcher.py
   '';
 
   postFixup = ''
+    # re-hardcode path to tail
+    sed -i 's|/usr/bin/tail|${coreutils}/bin/tail|' $out/lib/python2.7/site-packages/buildbot/scripts/logwatcher.py
```

Comment:
If you did this in the `patchPhase` or `pre/postPatch`, you wouldn't have to remove the .pyc file, right?